app/helpers/application_helper.rb

```diff
@@ -2,4 +2,10 @@ module ApplicationHelper
   def format_date_time(cert_date_time)
     cert_date_time.strftime("%e-%m-%Y %H:%M")
   end
+
+  def certificate_expiry(certificate)
```

Comment:
I think this should be a function should be part of the certificate. Generally when you see the entity reoccurring (in this certificate) this should give you an indication that operation belongs to that entity. Basically you want to be able to ask the certificate when it expires